lib/ansible/modules/storage/netapp/na_elementsw_access_group.py

```diff
@@ -299,10 +303,9 @@ def apply(self):
         update_group = False
 
         if self.account_id is not None:
-            self.account_id_valid = self.get_account_id()
-
+            self.account_id = self.get_account_id()
         if self.state == 'present' and self.volumes is not None:
-            if self.account_id_valid:
+            if self.account_id:
                 self.volumes = self.get_volume_id()
             else:
                 self.module.fail_json(msg='Error: Specified account id %s does not exist ' % self.account_id)
```

Comment:
The error message is now useless (or will generate a trace back) as self.account_id is now None.

Maybe we could add
input_account_id = str(self.account_id) before line 304.
and change the error message to:
self.module.fail_json(msg='Error: Specified account id %s does not exist ' % input_account_id)